interactive_engine/groot/src/main/java/com/alibaba/graphscope/groot/coordinator/NotifyFrontendListener.java

```diff
@@ -26,16 +27,19 @@
     private int frontendId;
     private FrontendSnapshotClient frontendSnapshotClient;
     private SchemaManager schemaManager;
+    private SnapshotCache snapshotCache;
 
     private AtomicLong lastDdlSnapshotId;
 
     public NotifyFrontendListener(
             int frontendId,
             FrontendSnapshotClient frontendSnapshotClient,
-            SchemaManager schemaManager) {
+            SchemaManager schemaManager,
+            SnapshotCache snapshotCache) {
```

Comment:
It's not appropriate to put snapshot cache in every listener, since only one snapshot cache is needed in coordinator but there are many listeners here.

Reply:
oka

Reply:
oka